app/templates/direct-award/_saved-search-locked-project-temp-message.html

```diff
@@ -0,0 +1,38 @@
+{% if project.lockedAt and following_framework %}
+  {% if temporary_message_status == 'locked_pre_live' %}
+    {% set temp_heading = 'The ' + '{}'.format(framework.name)|nbsp + ' services you found will expire soon' %}
+    {% set temp_messages = [
+        'You can either:',
+        ('<ul class="list-bullet-small"><li>award a contract for a '|safe + '{}'.format(framework.name)|nbsp +
+         ' service</li><li>start a new search for '|safe + '{}'.format(following_framework.name)|nbsp +
+         ' services from {}</li></ul>'.format(following_framework.frameworkLiveAtUTC|dateformat)|safe),
+        'Any existing contracts for ' + '{}'.format(framework.name)|nbsp + ' services are still valid.'
+      ]
+    %}
+  {% elif temporary_message_status == 'locked_post_live_during_interim' %}
+    {% set temp_heading = 'The ' + '{}'.format(framework.name)|nbsp + ' services you found will expire soon' %}
+    {% set temp_messages = [
+        'You can either:',
+        ('<ul class="list-bullet-small"><li>award a contract for a '|safe + '{}'.format(framework.name)|nbsp +
+         ' service</li><li><a href="{}">start a new search for '.format(buyer_search_page_url)|safe +
+         '{}'.format(following_framework.name)|nbsp + ' services</a></li></ul>'|safe),
+        'Any existing contracts for ' + '{}'.format(framework.name)|nbsp + ' services are still valid.'
+      ]
+    %}
+  {% elif temporary_message_status == 'locked_post_live_post_interim' %}
+    {% set temp_heading = 'The ' + '{}'.format(framework.name)|nbsp + ' services you found have expired' %}
+    {% set temp_messages = [
+        ('If you did not award a contract by {}, you need to <a href="{}">start a new search for '.format(framework.frameworkExpiresAtUTC|dateformat, buyer_search_page_url)|safe +
+        '{}'.format(following_framework.name)|nbsp + ' services</a>. If you did award a contract, please tell us the outcome.'|safe)
```

Comment:
`'{}'.format(following_framework.name)|nbsp ` can just be `following_framework.name|nbsp`, can't it?

Reply:
yes indeed.